### PR TITLE
executeRenameRefactoringBy-move-to-Variable

### PR DESCRIPTION
**Pull request by MarcusDenker**
move the executeRenameRefactoringBy: extensions methods from the RBVariableNode subclasses to the Variable hierarchy

### CODE REVIEW

**Comment by Ducasse** [comment]
We will have to think how to rename and where to place this nautilus refactoring.

File: src/NautilusRefactoring/GlobalVariable.extension.st [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Extension { #name : #GlobalVariable }
 
 { #category : #'*NautilusRefactoring' }
-GlobalVariable >> executeRenameRefactoringBy: aNautilusRefactoring inClass: aClass [
-	aNautilusRefactoring renameClass: value 
+GlobalVariable >> executeRenameRefactoringBy: aNautilusRefactoring [
+	aNautilusRefactoring renameClass: self definingClass
```
Comment (dionisiydk) [inline src/NautilusRefactoring/GlobalVariable.extension.st:5]: It looks like a bug. There is no definingClass for globals

Comment (guillep) [reply src/NautilusRefactoring/GlobalVariable.extension.st:5]: Probably because there are no tests about this.

Comment (MarcusDenker) [reply src/NautilusRefactoring/GlobalVariable.extension.st:5]: yes, but if we require 100% perfect tests before doing anything, we should stop now

**Pull request merged**